umh-core/.golangci.yaml

```
@@ -19,4 +19,32 @@ linters:
+    - nlreturn
+    - nolintlint
+    - perfsprint
+    - protogetter
+    - sloglint
+    - tagalign
+    - testifylint
+    - unconvert
+    - usestdlibvars
+    - usetesting
+    - whitespace
+    - wsl_v5
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)